python/amici/petab_import.py

```diff
@@ -174,51 +183,88 @@ def constant_species_to_parameters(sbml_model: 'libsbml.Model'
     return species_to_parameters(transformables, sbml_model)
 
 
-def import_model(sbml_file: str,
-                 condition_file: str,
-                 measurement_file: str = None,
-                 model_name: str = None,
+def import_model(sbml_model: Union[str, 'libsbml.Model'],
+                 condition_table: Optional[Union[str, pd.DataFrame]] = None,
+                 measurement_table: Optional[Union[str, pd.DataFrame]] = None,
+                 model_name: Optional[str] = None,
                  model_output_dir: str = None,
                  verbose: bool = True,
                  allow_reinit_fixpar_initcond: bool = False,
-                 **kwargs):
-    """Import AMICI model"""
+                 **kwargs) -> None:
+    """Create AMICI model from PEtab problem
+
+    Arguments:
+        sbml_model:
+            PEtab SBML model. If `libsbml.Model` is passed, this will be
+            modified inplace.
+        condition_table:
+            PEtab condition table. If provided, parameters from there will be
+            turned into AMICI constant parameters (i.e. parameter w.r.t. which
+            no sensitivities will be computed).
+        measurement_table:
+            PEtab measurement table
+        model_name:
+            Name of the generated model. If model file name was provided,
+            this defaults to the file name without extension, otherwise
+            the SBML model ID will be used.
+        model_output_dir:
+            Directory to write the model code to. Will be created if doesn't
+            exist. Defaults to current directory.
+        verbose:
+            Print/log extra information
+        allow_reinit_fixpar_initcond:
+            see amici.ode_export.ODEExporter
+        **kwargs:
+            Additional keyword arguments to be passed to
+            ``amici.sbml_importer.sbml2amici``.
+    """
 
+    # Color output
     init_colorama(autoreset=True)
 
+    if verbose:
```

Comment:
wouldn't it suffice to set the log level to debug somewhere? seems a bit double.

Reply:
True. This is inconsistent. Would try to clean that up with #884 and change to `logging` in more places.